cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -104,8 +128,10 @@ export default {
   },
   data() {
     return {
-      cards: [],
-      cjson: credentialDisplayData,
+      credentials: [],
+      genericCredentials: [],
+      manifestedCredentials: [],
```

Comment:
I am just thinking if we need two separate arrays after all?

Reply:
Yes for now see the description i have tried and already spent so much time to figure out to delete or splice the original array this is the simpler solution for now.

Reply:
No, I'm not saying we need to modify the original array. I'm thinking if we should just push both general and manifested credentials to the same array instead splitting them up in two?

Reply:
I dont see how to to do that we need to render both general and manifested credential at the same time and both have different thing to render on ui

Reply:
we need to render different ui container for the two scenario manifest and general, pushing them to the same array will not allow us to do that

Reply:
In the template, we could check the type of the credential from the array and render corresponding UI